contents/blog/posthog-vs-amplitude.mdx

```diff
@@ -16,57 +16,33 @@ import { ComparisonTable } from 'components/ComparisonTable'
 import { ComparisonHeader } from 'components/ComparisonTable/header'
 import { ComparisonRow } from 'components/ComparisonTable/row'
 
-Want to know how PostHog and Amplitude compare? If you remember nothing else, remember these two things:
-
-1. **Amplitude** is a product analytics tool built for non-technical users and data analysts.
-
-2. **PostHog** is an all-in-one platform built for engineers that replaces Amplitude _and_ numerous other tools.
-
 ## How is PostHog different?
 
-### 1. We're everything you need in one
+### 1. Engineering and product-led
 
-PostHog replaces multiple tools in your data stack. You could run Amplitude for analytics, [LaunchDarkly](/blog/posthog-vs-launchdarkly) for feature flags and testing, [Hotjar](/blog/posthog-vs-hotjar) for session replays and surveys, or you could just use PostHog for everything. One platform, one bill, and seamless integration.
+What does this mean?
 
-### 2. We're built for engineers
+- It means our pricing is 100% transparent. 
+- It means engineers make product decisions.
+- It means [we ship fast](/changelog) and iterate based on user feedback. 
+- It means engineers do support – all product teams have a [weekly support hero](/handbook/engineering/support-hero).
+- It means we work in the open – [our roadmap](/roadmap) and [company handbook](/handbook) are public.  
+- It means you can use PostHog without ever jumping on a "quick call" – see [how we do sales](/sales).
 
-We support all popular (and some unpopular) [SDKs](/docs/libraries), have [extensively documented APIs](/docs/api), and a [powerful SQL query builder](/docs/hogql), so you can analyze data how you want. We make it easy to [test in production](/product-engineers/testing-in-production), conduct [phased rollouts](/tutorials/phased-rollout), run [beta programs](/tutorials/public-beta-program), and so much more.
+### 2. Sustainably affordable
```

Comment:
I dont really like the word "sustainably" here. Makes me think of envirionment/climate stuff. Can we use a different word here and in the paragraph? Maybe something along the lines of "scalable"

Reply:
Some options:

- Affordable, usage-based pricing
- Affordable and transparent pricing
- Fair, usage-based pricing
- Transparent, usage-based pricing

Reply:
I think the second one

Reply:
I like the second one too